neqo-http3/src/hframe.rs

```diff
@@ -908,4 +935,213 @@ mod tests {
             panic!("wrong frame type");
         }
     }
+
+    fn test_reading_frame(
+        buf: &[u8],
+        send_fin_with_data: bool,
+        send_fin_after_data: bool,
+        expect_fin: bool,
+        succeed: bool,
+        frame_done: bool,
+    ) {
+        let (mut conn_c, mut conn_s) = connect();
+
+        // create a stream
+        let stream_id = conn_s.stream_create(StreamType::BiDi).unwrap();
+
+        let mut fr: HFrameReader = HFrameReader::new();
```

Comment:
```suggestion
        let mut fr = HFrameReader::new();
```

Also, you can move this down closer to where it is needed.